open_ce/inputs.py

```diff
@@ -251,6 +251,11 @@ class Argument(Enum):
                                      action='version',
                                      version="Open-CE Builder {}".format(open_ce_version)))
 
+    CONDA_PKG_FORMAT = (lambda parser: parser.add_argument(
```

Comment:
Should update the docs for build_env and build_config as well with the new option.

Reply:
Yes, I am yet to update the docs.  I will be updating them.